pkg/utils/etcdutil/health_checker.go

```diff
@@ -310,7 +310,7 @@ func (checker *healthChecker) updateEvictedEps(lastEps, pickedEps []string) {
 				zap.Int("picked-count-threshold", pickedCountThreshold),
 				zap.Int("picked-count", count.(int)+1),
 				zap.String("endpoint", ep),
-				zap.String("source", checker.source))
+				zap.String("source", string(checker.source)))
```

Comment:
It might not be a good idea to introduce a new type if it requires conversion every time we use it.

Reply:
yes, the construction function using the type but the inner field use the string type